src/orchestratord/src/controller/materialize/tls.rs

```diff
@@ -52,10 +52,10 @@ pub fn create_certificate(
             duration: mz_cert_spec.duration.or(default_spec.duration),
             issuer_ref,
             private_key: Some(CertificatePrivateKey {
-                algorithm: Some(CertificatePrivateKeyAlgorithm::Rsa),
+                algorithm: Some(CertificatePrivateKeyAlgorithm::Ed25519),
```

Comment:
This function is also used for balancers and the console, so this will cause compatibility issues.